src/main/java/org/spongepowered/common/mixin/core/entity/player/EntityPlayerMixin.java

```diff
@@ -310,9 +310,14 @@ private void postEventAndUpdateExperience(final int finalExperience) {
     }
 
     @Inject(method = "readEntityFromNBT", at = @At("RETURN"))
-    private void recalculateXpOnLoad(final NBTTagCompound compound, final CallbackInfo ci) {
+    private void onReadEntityFromNBT(final NBTTagCompound compound, final CallbackInfo ci) {
```

Comment:
I'm aware this is hypocritical (I myself started the trend...) but we're trying to move away to the "event-like" injector names. Instead, something like...

correctEntityPropertyMistakes? Dunno, names are hard. Just something better so that it is clear what the injector's purpose is.

Reply:
I'm honestly of the opinion of naming it appropriately: `impl$recalculateExpOnReadNBT`.

Reply:
Renamed to `impl$fixEntityProperties`